include/circt/Dialect/RTL/RTLStructure.td

```diff
@@ -19,7 +19,8 @@ def RTLModuleOp : RTLOp<"module",
     name, a list of ports, and a body that represents the connections within
     the module.
   }];
-  let arguments = (ins StrArrayAttr:$argNames, StrArrayAttr:$resultNames);
```

Comment:
Please split stuff like this out to its own patch since it is logically separate.

That said, I don't think we need this - the reason that extmodule has this is because you can have multiple extmodule instances with the same verilog name (due to parameters).  RTLModuleOp doesn't have that going on.